src/knx/device_object.h

```diff
@@ -44,8 +50,11 @@ class DeviceObject: public InterfaceObject
     uint8_t _prgMode = 0;
     uint16_t _ownAddress = 0;
     uint16_t _manufacturerId = 0xfa; //Default to KNXA
-    uint32_t _bauNumber = 0;
+    uint32_t _bauNumber = 0xaabbccdd;
```

Comment:
Is there a reason why this can't stay zero on default?

Reply:
Yes, the reason was debugging. Have zero everywhere sometimes hides the fact that not enough bytes were copied for example. Easpecially when included in payload of an APDU, so at the end.
I can remove it again.

Reply:
Please do.

Reply:
done.